src/ORM/Query.php

```diff
@@ -30,7 +31,7 @@
  * required.
  *
  */
-class Query extends DatabaseQuery implements JsonSerializable
+class Query extends DatabaseQuery implements JsonSerializable, QueryInterface
```

Comment:
Yes we can but some methods in the ```Query``` class have comments more specific to their usage in a database table. Like the support for ```AND``` and ```OR```.